website/pages/playground/index.html

```diff
@@ -137,8 +137,8 @@ <h1>Prettier <span id="version"></span></h1>
         <details class="sub-options">
           <summary>Range</summary>
           <label>The selected range will be highlighted in yellow in the input editor</label>
-          <label>--range-start <input type="number" min="0" id="rangeStart"></label>
-          <label>--range-end <input type="number" min="0" id="rangeEnd"></label>
+          <label>--range-start <input type="number" value="" min="0" id="rangeStart"></label>
+          <label>--range-end <input type="number" value="" min="0" id="rangeEnd"></label>
```

Comment:
To exclude unnecessary options while clicking <kbd>copy markdown</kbd>:

```
--range-start
--range-end
```

https://github.com/prettier/prettier/blob/2e73164b81cc165ab55c957ead7b7c9a957c9d69/website/static/playground.js#L326-L330